lib/license_finder/package_managers/pip.rb

```diff
@@ -55,7 +55,7 @@ def pip_output
     end
 
     def pypi_def(name, version)
-      response = HTTParty.get("https://pypi.python.org/pypi/#{name}/#{version}/json")
+      response = HTTP.follow.get("https://pypi.python.org/pypi/#{name}/#{version}/json")
```

Comment:
I don't think the JSON will parse correctly like this since `response.body` [returns a body object](https://github.com/httprb/http/wiki/Response-Handling).

You could call `.parse` directly on the response or call `.to_s` on the response before `JSON.parse`

Reply:
Actually it does work (I verified), but you might be right that using `.parse` is cleaner.

Reply:
hah, TIL 👏